site/content/docs/5.3/components/toasts.md

```diff
@@ -172,10 +172,12 @@ Alternatively, you can also add additional controls and components to toasts.
 
 ### Color schemes
 
-Building on the above example, you can create different toast color schemes with our [color]({{< docsref "/utilities/colors" >}}) and [background]({{< docsref "/utilities/background" >}}) utilities. Here we've added `.text-bg-secondary` to the `.toast`. For a crisp edge, we remove the default border with `.border-0`.
+Building on the above example, you can create different toast color schemes with our [color]({{< docsref "/utilities/colors" >}}) and [background]({{< docsref "/utilities/background" >}}) utilities. Here we've added `.text-bg-dark` to the `.toast`. For a crisp edge, we remove the default border with `.border-0`.
+
+{{< callout-deprecated-dark-variants "toast" >}}
```

Comment:
This wasn't really a dark variant but I can understand why you're adding this here. However, in this case, we shouldn't need to use `.text-bg-dark` isn't it?

Reply:
We shouldn't but since we don't have the `data-bs-theme="inverted"` or something similar, we won't be able to use `.text-bg-secondary` when it's including other components. so to keep the same spirit at least in light theme, I went this way.